.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,40 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      
+      - name: Log in to Docker Hub
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+        with:
+          images: fedcampus/dyn_flower_android_drf
+      
+      - name: Build and push Docker image
+        uses: docker/build-push-action@3b5e8027fcad23fda98b2e3ac259d8d67585f671
+        with:
+          context: ./backend
+          file: ./backend/Dockerfile
```

Comment:
The problem here is that `Dockerfile` runs `run.sh`, which uses `seed.py` to populate the database with the CIFAR10 data type and model, but the model is not downloaded.

My take is that the we either do 1. or 2.:

1. Do not download the model and do not seed the database.
2. Download the model and seed the database.

Reply:
> This design is to allow for different models to be passed through to the container without the need of rebuilding the image.

If so, I think the Dockerfile should not use `seed.py` because it injects the model information into the database, which would mess up the container if one were to use other models.

Reply:
My understanding of the `seed.py` in the future is that it will automatically add all the models to the docker image at startup.
Additionally, even if the seed.py needs to be different, it can be replaced with a bind mount

Reply:
The way to persist database across different images is to bring the database out and modify the way to initialize the db. I believe have left enough room for this

Reply:
Another option with `seed.py` is that we can use two environment variables to control if we'll use the builtin `seed.py`, if not, the other env can be used to passthrough the custom command that generates the content that will be piped through to the `manage.py console`